Format check complains about deleted files

If I delete a file and commit the deletion, the format check will complain about the file having the wrong format:
```
$ git checkout -b clang-format-ignore-deleted-files
$ git rm src/libs/syncpoint/syncpoint.cpp 
rm 'src/libs/syncpoint/syncpoint.cpp'
$ git status
## clang-format-ignore-deleted-files
D  src/libs/syncpoint/syncpoint.cpp
$ make quick-check
[CHK] Checking configuration files
[CHK] Checking license headers
[CHK] Checking code formatting in branch and modified files
--> All source files properly formatted
--> No warnings. Nice job.
--> All source files have a proper license header.
$ git commit -m "test removal"
[clang-format-ignore-deleted-files f0e261322] test removal
 1 file changed, 631 deletions(-)
 delete mode 100644 src/libs/syncpoint/syncpoint.cpp
$ make quick-check
[CHK] Checking code formatting in branch and modified files
0% 0:1=0s src/libs/syncpoint/syncpoint.cpp                                                               No such file or directory
100% 1:0=0s src/libs/syncpoint/syncpoint.cpp                                                             
--> src/libs/syncpoint/syncpoint.cpp is not properly formatted
--> 1/1 have bad formatting.
make: *** [/home/thofmann/code/fawkes/etc/buildsys/root/check.mk:73: format-check-branch] Error 1
```

--- a/etc/format-scripts/check-file.sh
+++ b/etc/format-scripts/check-file.sh
@@ -18,6 +18,10 @@ set -eu
 
 FILE=$1
 
+if [ ! -e $FILE ] ; then
+	exit 0
+fi
+
 if ! cmp -s <(clang-format $FILE) $FILE; then
 	echo $FILE
 fi

